store/opentracing_layer.go

```diff
@@ -465,11 +445,6 @@ func (s *OpenTracingLayerBotStore) Update(bot *model.Bot) (*model.Bot, *model.Ap
 
 	defer span.Finish()
 	resultVar0, resultVar1 := s.BotStore.Update(bot)
-	if resultVar1 != nil {
```

Comment:
This is a symptom that the opentracing generator is not capturing errors, only `model.AppError`, probably we need to modify the generator to tackle that. Is it correct @reflog?

Reply:
@jespino - I'll take a look and report back

Reply:
yet. happens due to:
```go

		"errorPresent": func(results []string) bool {
			for _, typeName := range results {
				if strings.Contains(typeName, APP_ERROR_TYPE) {
					return true
				}
			}
			return false
		},
```
I'll submit the PR to fix

Reply:
Thanks @reflog !